### ISSUE TITLE
Issue getting mosvis to work w astroconda

### ISSUE THREAD

**Issue by janerigby** [spacetelescope/mosviz#55]
Just downloaded astroconda for the Proposal Planning workshop, and on a lark tried to load the JWST Data Analysis tools.  It refuses to import mosviz, with the following error:
```python
In [4]: import mosviz
INFO:glue:Loading settings from /Users/jrrigby1/.glue/settings.cfg
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-329a497a30b0> in <module>()
----> 1 import mosviz

/Applications/anaconda/envs/iraf27/lib/python2.7/site-packages/mosviz/__init__.py in <module>()
     16 if not _ASTROPY_SETUP_:
     17     from . import loaders
---> 18     from . import viewers
     19     from . import widgets
     20 

/Applications/anaconda/envs/iraf27/lib/python2.7/site-packages/mosviz/viewers/__init__.py in <module>()
      1 from __future__ import absolute_import
      2 
----> 3 from .mos_viewer import *

/Applications/anaconda/envs/iraf27/lib/python2.7/site-packages/mosviz/viewers/mos_viewer.py in <module>()
     11 from ..widgets.viewer_options import OptionsWidget
     12 
---> 13 from glue.qt.widgets.data_viewer import DataViewer
     14 from glue import config
     15 from glue.core import message as msg

ImportError: No module named qt.widgets.data_viewer
```

**Comment by astrofrog** [comment]
MOSViz needs to be updated to support the latest version of glue - I'll work on it now

**Comment by astrofrog** [comment]
Fixed by #56
